.github/workflows/push-and-pr-testing-ruby-ci.yml

```diff
@@ -15,6 +15,7 @@ jobs:
     strategy:
       matrix:
         container_tag:
+        - 3.2-alpine
         - 3.1-alpine
         - 3.0-alpine
         - 2.7-alpine
```

Comment:
ruby 2.6 is EOL and support for 2.7 will be discontinued in April. I recommend removing them at some point

Reply:
We can update to release 3.1 and drop support for ruby < 3

Reply:
Would like me to update that in this PR?

Reply:
No, I will do this on merge, please